firstvoices/backend/permissions/predicates/base.py

```diff
@@ -98,10 +98,16 @@ def is_superadmin(user, obj):
     return get_app_role(user) == AppRole.SUPERADMIN
 
 
+@predicate
+def is_hidden_obj(user, obj):
+    return obj.is_hidden
```

Comment:
Will this cause errors if this predicate is used on a model that doesn't have an is_hidden field? If so, I think the name needs to be more specific, like is_hidden_site